src/App.tsx

```diff
@@ -15,9 +17,12 @@ if (BASE_URL === '') {
 const userRepository = new UserJSDataverseRepository()
 function App() {
   return (
-    <SessionProvider repository={userRepository}>
-      <Router />
-    </SessionProvider>
+    <>
+      <SessionProvider repository={userRepository}>
+        <Router />
+      </SessionProvider>
+      <ToastContainer position="top-right" autoClose={5000} pauseOnHover />
```

Comment:
I think the progress bar is a little confusing because it implies that the update action is still in process, what do you think?  Maybe we can have it display only for long running processes.
```suggestion
      <ToastContainer hideProgressBar={true}  position="top-right" autoClose={5000} pauseOnHover />
```

Reply:
That's ok, when the toast appears the action was already done, that's why the text says 'sucessfully updated' or 'successfully deleted', the progress bar is only to show when the toast will go away.

Reply:
We can remove it if you think is better yes, but for long running processes will not add any value because as I said, when the toast appears the action was already completed.

Reply:
ok, I see what you mean that it's not needed for long running processes. Maybe it's best to leave it for now and see what others think, it's easy to change later 👍

Reply:
yes, is really easy to change and also those are global settings but you can override them from every individual toast you trigger